server/package.json

```diff
@@ -54,25 +54,26 @@
     "unionized": "^4.12.0"
   },
   "scripts": {
-    "coverage":
-      "nyc --check-coverage --lines 90 --reporter=html --reporter=text --reporter=lcov --extension .es6 mocha ./test/**/*.es6 ./test/*.es6",
+    "coverage": "nyc --check-coverage --lines 90 --reporter=html --reporter=text --reporter=lcov --extension .es6 mocha ./test/**/*.es6 ./test/*.es6",
     "dev": "yarn; sequelize db:migrate; nodemon --harmony_default_parameters ./src/app.es6",
-    "lint":
-      "eslint ./app.es6 ./models/noncommercial-application.es6 ./util.es6 ./test/noncommercial.js ./middlelayer-interaction.es6 ./test/test-util.js ./test/middlelayer-unit.js ./validation.es6 ./test/test-validate.js ./vcap-services.es6 ./temp-outfitter.es6 ./test/tempoutfitter-tests.js ./noncommercial.es6 ./application-general.es6",
+    "lint": "eslint ./app.es6 ./models/noncommercial-application.es6 ./util.es6 ./test/noncommercial.js ./middlelayer-interaction.es6 ./test/test-util.js ./test/middlelayer-unit.js ./validation.es6 ./test/test-validate.js ./vcap-services.es6 ./temp-outfitter.es6 ./test/tempoutfitter-tests.js ./noncommercial.es6 ./application-general.es6",
     "migrate": "sequelize db:migrate",
-    "report-coverage":
-      "CODECOV_TOKEN='e61f7ed1-686a-4f48-95b3-02fa7085995e' ./node_modules/nyc/bin/nyc.js report --reporter=lcov > coverage.lcov && codecov",
-    "seed": "sequelize db:seed:all",
+    "report-coverage": "CODECOV_TOKEN='e61f7ed1-686a-4f48-95b3-02fa7085995e' ./node_modules/nyc/bin/nyc.js report --reporter=lcov > coverage.lcov && codecov",
```

Comment:
for open repos you do not need a token. Please remove this key.

Reply:
@lauraGgit codecov reports are only for server code and they do not match either. We plan to drop this and continue to use the coverage reports from CircleCI artifacts.